R/pkg/R/mllib.R

```diff
@@ -57,6 +57,9 @@ setClass("KMeansModel", representation(jobj = "jobj"))
 #'
 #' Saves the MLlib model to the input path. For more information, see the specific
 #' MLlib model below.
+#' @param object a fitted ML model object.
+#' @param path the directory where the model is saved.
+#' @param ... additional argument(s) passed to the method.
```

Comment:
this is more of a placeholder page to direct to other pages. there's no function signature on this output page and so I don't think we should add parameter here either.